### PR TITLE
feat: include metabolomics in condition data response

### PR DESCRIPTION
**Pull request by kvikshaug**

### CODE REVIEW

**Review by annasirunian — APPROVED** [review]
Looks good

File: src/warehouse/models.py [hunk 1]
```diff
@@ -175,3 +175,11 @@ def is_fluxomics(self):
             and self.numerator.type.name == "reaction"
             and self.denominator is None
         )
+
+    def is_metabolomics(self):
+        return (
+            self.numerator is not None
+            and self.numerator.type.name == "compound"
+            and self.denominator is None
```
Comment (ChristianLieven) [inline src/warehouse/models.py:183]: When the denominator is none we automatically assume it is per growth?

I'm just trynig to understand :)

Comment (ChristianLieven) [reply src/warehouse/models.py:183]: If the denominator was another compound we'd get a ratio or yield I suppose?

**Review by ChristianLieven — APPROVED** [review]

File: src/warehouse/models.py (hunk 1 above)
Comment (kvikshaug) [reply src/warehouse/models.py:183]: Yes indeed, then we would use the `compound / compound (Cmol/Cmol)` unit. The warehouse does accomodate that type of data, but integration is not yet implemented on the modeling side.

**Pull request merged**